README.md

```diff
@@ -1,2 +1,27 @@
-# bpmn-visualization-R
+<h1 align="center">BPMN Visualization - R Package</h1>
+<div align="center">
+    <p align="center">
+        <a href="https://github.com/process-analytics/bpmn-visualization-r/releases">
+          <img alt="GitHub release (latest by date including pre-releases)" src="https://img.shields.io/github/v/release/process-analytics/bpmn-visualization-r?label=changelog&include_prereleases"> 
+        </a>
+        <a href="https://github.com/process-analytics/bpmn-visualization-r/actions">
+          <img alt="Build" src="https://github.com/process-analytics/bpmn-visualization-r/workflows/Build/badge.svg"> 
+        </a> 
+        <a href="https://gitpod.io/#https://github.com/process-analytics/bpmn-visualization-r" target="_blank">
+          <img alt="Gitpod" src="https://img.shields.io/badge/Gitpod-ready--to--code-chartreuse?logo=gitpod"> 
+        </a> 
+        <br>
+        <a href="CONTRIBUTING.md">
+          <img alt="PRs Welcome" src="https://img.shields.io/badge/PRs-welcome-ff69b4.svg?style=flat-square"> 
+        </a> 
+        <a href="CODE_OF_CONDUCT.md">
+          <img alt="Contributor Covenant" src="https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg"> 
+        </a> 
+        <a href="LICENSE">
+          <img alt="License" src="https://img.shields.io/github/license/process-analytics/bpmn-visualization-r?color=blue"> 
+        </a>
+    </p>
+</div>  
+<br>
+
 A R package which embeds BPMN Visualization 
```

Comment:
To be improved in the future: bpmn-visualization is the way used by the package to display execution data over BPMN Diagrams